### PR TITLE
Official Meteor integration

### PR DESCRIPTION
**Pull request by splendido**
Hello guys,

As you might know, the [Meteor](https://www.meteor.com/) community is widely using Bootstrap. We had many different wrapper packages all doing the same thing, so I'm submitting a PR that integrates Meteor packaging directly into your repository: at the end of this merge process, in case you'll find yourself comfortable with it, we'll get automated new meteor package releases at every new release of Bootstrap.

I'm aware of the [previous attempt](https://github.com/twbs/bootstrap/pull/15376) made by @dandv, but I've made this one as lean as possible so not to pick up too much space within your repository: basically it adds only a `pacakge.js` file among the ones already present for the other PMs. This should be not too much pretending.

All you have to do after accepting this PR would be:
1. create an account at https://meteor.com/ (click SIGN IN, then Create account). After you've done that, please let me know the name of the account, and we'll add you as a maintainer of the `twbs` organization (which has been alredy reserved to make things simpler for you).
2. head to [autopublish.meteor.com](http://autopublish.meteor.com/) sign in with your gitHub account, locate this repository and enable it for autopublish by toggling the checkbox you'll find on the right side of the repository item. You can get an idea about how Meteor Autopublish works by having a quick look at [this page](http://autopublish.meteor.com/howitworks), but basically it creates a [web hook](https://help.github.com/articles/about-webhooks/) to let us know when you push new tags. And when you'll release new versions, autopublish.meteor.com will automatically take delivery for publishing the new version also for Meteor!

Please note that you will not be able to toggle your repository until you'll have the `package.js` file on your master branch.

Please also note that we've already published the current version of the package on [Atmosphere](https://atmospherejs.com/twbs/bootstrap) (Meteor's package directory) under the name `twbs:bootstrap`. 
But if you think another name would be a better fit, just let us know!

Thanks in advance for your availability and collaboration and congratulation for the awesome project!
Luca & the [Meteor integrations team](https://github.com/raix/Meteor-community-discussions/issues/14)

### CODE REVIEW

File: package.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+Package.describe({
+  name: 'twbs:bootstrap',  // http://atmospherejs.com/twbs/bootstrap
+  summary: 'The most popular front-end framework for developing responsive, mobile first projects on the web.',
+  version: '3.3.2',
+  git: 'https://github.com/twbs/bootstrap.git'
+});
+
+Package.onUse(function (api) {
+  api.versionsFrom('METEOR@1.0');
+	api.use('jquery');
```
Comment (cvrebert) [inline package.js]: Why is this line indented?

File: package.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+Package.describe({
+  name: 'twbs:bootstrap',  // http://atmospherejs.com/twbs/bootstrap
+  summary: 'The most popular front-end framework for developing responsive, mobile first projects on the web.',
+  version: '3.3.2',
+  git: 'https://github.com/twbs/bootstrap.git'
+});
+
+Package.onUse(function (api) {
+  api.versionsFrom('METEOR@1.0');
+	api.use('jquery', 'client');
```
Comment (splendido) [inline package.js]: This line asks Meteor to let this package see the `jquery` package (a core Meteor package actually) which exports the jQuery object [you need in your `dist/js/bootstrap.js`](https://github.com/twbs/bootstrap/blob/master/dist/js/bootstrap.js#L7).

...I've just updated it to better specify that the requirement is only for client code.

Comment (cvrebert) [inline package.js]: Okay, but why is it indented more than the previous and subsequent lines?

Comment (splendido) [inline package.js]: oh, sorry, I've read 'intended'...

**Comment by cvrebert** [comment]
As this version is basically akin to the existing Bower and Composer packagings (i.e. add one metadata file and a webhook), I'm +0.5 on this.

**Comment by mdo** [comment]
I'm for it as well.

@cvrebert Will our Grunt-based version updater address this file's version number as well, or do we need to do it manually?

**Comment by cvrebert** [comment]
It'll be automatic. The updater is overzealous, not underzealous.

**Comment by splendido** [comment]
@cvrebert we'll have to wait for @dandv, he originally created the org and I have no access to it :(

**Comment by dandv** [comment]
Sorry guys, didn't see this. I've added both cvrebert and splendido to the MD org. Thanks for all your work!

**Comment by cvrebert** [comment]
Okay, I added the webhook. So, we're all good now, right?

**Comment by splendido** [comment]
Yep!
Everything seems fine and ready for the autopublish at the next release :)

Thank you guys,
Keep up the awesome work!

**Comment by splendido** [comment]
Eventually we had the proof that everything is set up correctly to publish new versions of the `twbs:bootstrap` meteor package within minutes from new releases!
See it on [autopublish](http://autopublish.meteor.com/) :-)

Thanks again everyone!
Cheers,
Luca

**Pull request closed without merging**